<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintDialog</class>
 <widget class="QWidget" name="PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrintDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Printer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="printerChoice">
       <item>
        <property name="text">
         <string>Canon InkWaster XYZ3000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sister PrintMaster 800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HP PaperJammer B200</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="groupLayout">
     <item>
      <widget class="QGroupBox" name="orientationGroup">
       <property name="title">
        <string>Page Orientation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioBtnLandscape">
          <property name="text">
           <string>Landscape</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBtnPortrait">
          <property name="text">
           <string>Portrait</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="colorGroup">
       <property name="title">
        <string>Color Options</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioBtnBlackWhite">
          <property name="text">
           <string>Black and White</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBtnColor">
          <property name="text">
           <string>Color</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPrint">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
